ui/client/components/right-panel/UserRightPanel.js

```diff
@@ -119,7 +119,7 @@ class UserRightPanel extends Component {
         panelName: "Deployment",
         buttons:[
           {name: "deploy", visible: this.props.capabilities.deploy, disabled: !conf.deployPossible, icon: InlinedSvgs.buttonDeploy, btnTitle: conf.deployToolTip, onClick: this.deploy, onMouseOver: conf.deployMouseOver, onMouseOut: conf.deployMouseOut},
-          {name: "stop", visible: this.props.capabilities.deploy, disabled: !this.isRunning(), onClick: this.stop, icon: InlinedSvgs.buttonStop},
+          {name: "cancel", visible: this.props.capabilities.deploy, disabled: !this.isRunning(), onClick: this.stop, icon: InlinedSvgs.buttonStop},
```

Comment:
maybe let's change also method and button name to avoid confustion?

Reply:
You're right, done